packages/e2e/cypress/support/commands.ts

```diff
@@ -77,9 +84,93 @@ Cypress.Commands.add('registerNewUser', () => {
     });
 });
 
+Cypress.Commands.add(
+    'registerWithCode',
+    (email: string, inviteCode: string) => {
+        cy.request({
+            url: `api/v1/user`,
+            headers: { 'Content-type': 'application/json' },
+            method: 'POST',
+            body: {
+                inviteCode,
+                email,
+                firstName: 'test',
+                lastName: 'test',
+                password: 'test',
+            },
+        }).then((resp) => {
+            cy.log(JSON.stringify(resp.body));
+            expect(resp.status).to.eq(200);
+        });
+    },
+);
+
+Cypress.Commands.add('invite', (email: string, role: string) => {
+    const expiresAt = new Date(Date.now() + 24 * 60 * 60 * 1000); // in 1 day
+
+    cy.request({
+        url: `api/v1/invite-links`,
+        headers: { 'Content-type': 'application/json' },
+        method: 'POST',
+        body: {
+            role,
+            email,
+            expiresAt,
+        },
+    }).then((resp) => {
+        cy.log(JSON.stringify(resp.body.results));
+        expect(resp.status).to.eq(201);
+        cy.wrap(resp.body.results.inviteCode);
+    });
+});
+
+Cypress.Commands.add(
+    'addProjectPermission',
+    (email: string, role: string, projectUuid: string) => {
+        cy.request({
+            url: `api/v1/projects/${projectUuid}/access`,
+            headers: { 'Content-type': 'application/json' },
+            method: 'POST',
+            body: {
+                role,
+                email,
+            },
+        }).then((resp) => {
+            expect(resp.status).to.eq(200);
+        });
+    },
+);
+
 Cypress.Commands.add('logout', () => {
     cy.request({
         url: 'api/v1/logout',
         method: 'GET',
     });
 });
+
+type ProjectPermission = {
+    role: string;
+    projectUuid: string;
+};
+Cypress.Commands.add(
+    'loginWithPermissions',
```

Comment:
New command to : 
- invite user with X role 
- add X role to project 
- register with invitation code